opentracing-examples/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenTracing-Java examples
+
+Examples of common instrumentation patterns:
+
+- **activate_deactivate** - callbacks finish at some time.   
```

Comment:
Could we create some categories? e.g. one related to client-server other to callbacks and order them from the simplest to more complex use cases?